docs/reference/icalendar.alarms.rst

```diff
@@ -0,0 +1,7 @@
+icalendar.alarms module
```

Comment:
My thought: Open an issue to create tests that check that all .py files have a corresponding .rst file.

Reply:
#904. Feel free to comment or merge.